cypress/e2e/integration/MainPage.cy.ts

```diff
@@ -0,0 +1,98 @@
+import { DateProps } from "interfaces/date_props.interface";
+
+describe("Renders Main page of app", () => {
+  const lengthOfItems = 5;
+  it("Renders correctly", () => {
+    cy.visit("/");
+  });
+
+  it("Testing date, Checking if the current date that shown on dom is the same like I created over here", () => {
+    const today = new Date(Date.now());
+    const convertedDate: DateProps = {
+      day: today.getDate(),
+      month: today.toLocaleString("default", { month: "long" }),
+      year: today.getFullYear(),
+      currentDay: today.toLocaleString("default", { weekday: "long" }),
+    };
+    cy.findByTestId("header-day").should("have.text", convertedDate.day);
+    cy.findByTestId("header-month").should("have.text", convertedDate.month);
+    cy.findByTestId("header-year").should("have.text", convertedDate.year);
+    cy.findByTestId("header-day-name").should(
+      "have.text",
+      convertedDate.currentDay
+    );
+  });
+
+  it("Testing CRUD on 1 Task", () => {
+    cy.findByTestId("task-input-field").type("Item 1");
+    cy.findByTestId("create-task").click();
+    cy.findByTestId("created-tasks-list").children().should("have.length", 1);
```

Comment:
I don't like that the number of tasks is explicit. Maybe calculate and make it dynamic